frontend/app/.ic-assets.json5

```diff
@@ -8,7 +8,7 @@
             "Strict-Transport-Security": "max-age=31536000; includeSubDomains",
             "X-Content-Type-Options": "nosniff",
             "X-XSS-Protection": "1; mode=block",
-            "Content-Security-Policy": "frame-ancestors https://4hy5z-qaaaa-aaaal-addrq-cai.icp0.io https://desktop.windoge98.com https://vault-bet.com https://www.vault-bet.com https://gooble.app  https://signalsicp.com https://kjfeq-waaaa-aaaan-qedva-cai.icp0.io https://spyzr-gqaaa-aaaan-qd66q-cai.icp0.io https://221bravo.app https://ht7v7-iaaaa-aaaak-qakga-cai.icp0.io https://mdocx-gyaaa-aaaak-qcbsq-cai.icp0.io https://calm-pasca-49d7be.netlify.app http://localhost:5173 https://zexzi-jyaaa-aaaam-abj3q-cai.icp0.io https://xw4dq-4yaaa-aaaam-abeuq-cai.ic0.app https://bxnm7-oqaaa-aaaag-albpq-cai.icp0.io https://panoramablock.com;",
+            "Content-Security-Policy": "frame-ancestors https://4hy5z-qaaaa-aaaal-addrq-cai.icp0.io https://desktop.windoge98.com https://vault-bet.com https://www.vault-bet.com https://gooble.app  https://signalsicp.com https://kjfeq-waaaa-aaaan-qedva-cai.icp0.io https://spyzr-gqaaa-aaaan-qd66q-cai.icp0.io https://221bravo.app https://ht7v7-iaaaa-aaaak-qakga-cai.icp0.io https://mdocx-gyaaa-aaaak-qcbsq-cai.icp0.io https://calm-pasca-49d7be.netlify.app http://localhost:5173 https://zexzi-jyaaa-aaaam-abj3q-cai.icp0.io https://xw4dq-4yaaa-aaaam-abeuq-cai.ic0.app https://bxnm7-oqaaa-aaaag-albpq-cai.icp0.io https://panoramablock.com https://dragonwizards.club/lair.html;",
```

Comment:
I think you need to strip off the `/lair.html`.
@julianjelfs will know better though.

Reply:
Yes needs to be just the origin i.e. `https://dragonwizards.club`